### PR TITLE
chore: add Chuks

### PR DESCRIPTION
**Pull request by ChuksMbanaso**
# Add Chuks

##What does this PR do?
```
Add my profile 

```

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/925"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `chore: add Chuks`

File: public/data/Chuksmbanaso.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Chuks",
+  "type": "personal",
+  "bio": "Fullstack Developer | Open Source Enthusiast",
+  "avatar": "https://github.com/Chuksmbanaso.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://linkfree.eddiehub.org/Chuksmbanaso",
```
Comment (syedsohan) [inline public/data/Chuksmbanaso.json:9]:
```suggestion
      "url": "https://github.com/ChuksMbanaso",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/Chuksmbanaso.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Chuks",
+  "type": "personal",
+  "bio": "Fullstack Developer | Open Source Enthusiast",
+  "avatar": "https://github.com/Chuksmbanaso.png",
```
Comment (krishguptadev) [inline public/data/Chuksmbanaso.json:5]:
```suggestion
  "avatar": "https://github.com/ChuksMbanaso.png",
```

File: public/data/Chuksmbanaso.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Chuks",
+  "type": "personal",
+  "bio": "Fullstack Developer | Open Source Enthusiast",
```
Comment (Panquesito7) [inline public/data/Chuksmbanaso.json:4]:
```suggestion
  "bio": "Full-stack Developer | Open Source Enthusiast",
```

File: public/data/Chuksmbanaso.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "Follow me on GitHub",
+      "url": "http://linkfree.eddiehub.org/Chuksmbanaso",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Chuksmbanaso",
+      "icon": "twitter"
+    }
+  ],
```
Comment (syedsohan) [inline public/data/Chuksmbanaso.json:17]:
```suggestion
  ]
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Comment by emmalearnscode** [comment]
> Change the PR title to - `chore: add Chuks`

@syedsohan Please can you leave this to the maintainers to change, it's fine for new contributors to choose the title themselves :)

**Comment by syedsohan** [comment]
> > Change the PR title to - `chore: add Chuks`
> 
> @syedsohan Please can you leave this to the maintainers to change, it's fine for new contributors to choose the title themselves :)

Ok. 🙂

**Comment by KendallDoesCoding** [comment]
I dont think @ChuksMbanaso is too active on github?

File: public/data/Chuksmbanaso.json (hunk 4 above)
Comment (KendallDoesCoding) [reply public/data/Chuksmbanaso.json:17]: I think, github-actions can do this, so dw about it :)

**Comment by emmalearnscode** [comment]
@ChuksMbanaso ill close this pull request for now. Feel free to open it again if you wish to continue working on it.

File: public/data/Chuksmbanaso.json (hunk 3 above)
Comment (eddiejaoude) [reply public/data/Chuksmbanaso.json:4]: This is a personal preference, I have seen it written many ways and I also write it as 1 word `fullstack`

**Pull request closed without merging**